doc/upgrade/7.2.md

```diff
@@ -31,3 +31,24 @@ For legacy, in `ezpublish_legacy/settings/site.ini`, set the following:
 [DatabaseSettings]
 Charset=utf8mb4
 ```
+
+## Notifications moved to Community edition
+
+Following query should be executed on database in order to add new table
```

Comment:
This is not how we do it. Please place proper upgrade scripts in:
- `./data/update/mysql/`
- `./data/update/postgres/`

Of course then you can link these files here.